ui-meta/slurm-infra.yml

```diff
@@ -28,6 +28,15 @@ parameters:
     options:
       min_ram: 2048
       min_disk: 10
+  
+  - name: home_volme_size
+    label: Home volume size
+    description: The size in GB of the volume to use for home directories
+    kind: integer
+    immutable: true
```

Comment:
Its a shame that we can't guarantee online resize works, it would be cool to make this mutable so that people could extend volumes when they got full!